src/Trace/Integrations/PDO.php

```diff
@@ -31,19 +31,18 @@
  */
 class PDO implements IntegrationInterface
 {
-    static $dsn = "";
+    static $params = [];
     /**
      * Static method to add instrumentation to the PDO requests
      */
-    public static function load($dsn="")
+    public static function load($params = [])
```

Comment:
@akash let's document what can come in params and how they are used. $dsn was clearer in that sense.
imo, it's better to take $dsn explicitly and optional params.